pkg/kontainer-engine/cmd/ls.go

```diff
@@ -27,7 +27,7 @@ func lsCluster(ctx *cli.Context) error {
 	writer := utils.NewTableWriter([][]string{
 		{"NAME", "Name"},
 		{"DRIVER", "DriverName"},
-		{"VERISON", "Version"},
+		{"VERSION", "Version"},
```

Comment:
This whole cli probably just needs to be deleted but outside the scope of this PR 😄